### ISSUE TITLE
Nx Console generation commands do nothing (but spin continuously)

### ISSUE THREAD

**Issue by mwarger** [nrwl/nx-console#1291]
## Current Behavior

Clicking on the console and generate option results in the form being filled out.  The console attempts to 'dry run' the appropriate commands, but outputs nothing and the command just "spins".  Clicking 'Run' does nothing.  See screenshots.

## Expected Behavior

This should work like it used to.

## Steps to Reproduce

This started happening maybe a week ago?  I'm really not sure.

### Failure Logs / Images / Videos

<img width="991" alt="image" src="https://user-images.githubusercontent.com/686823/172286597-4991e79a-5dbe-4ce2-8489-3604d4081983.png">

Red box on left: Command outputs nothing while...

Red box on right: The console process is spawned and spins continuously.  Unlike normal use of the NX console where it says you can press a key to close the terminal, this one is allowed to be used.  The spinning never stops.

### Environment

❯ nx report

 >  NX   Report complete - copy this into the issue template

   Node : 16.15.0
   OS   : darwin x64
   yarn : 1.22.19
   
   nx : 14.1.7
   @nrwl/angular : Not Found
   @nrwl/cypress : 14.1.7
   @nrwl/detox : 14.1.9
   @nrwl/devkit : 14.1.7
   @nrwl/eslint-plugin-nx : 14.1.7
   @nrwl/express : 14.1.7
   @nrwl/jest : 14.1.7
   @nrwl/js : 14.1.7
   @nrwl/linter : 14.1.7
   @nrwl/nest : Not Found
   @nrwl/next : Not Found
   @nrwl/node : 14.1.7
   @nrwl/nx-cloud : 14.0.5
   @nrwl/nx-plugin : Not Found
   @nrwl/react : 14.1.7
   @nrwl/react-native : Not Found
   @nrwl/schematics : Not Found
   @nrwl/storybook : 14.1.7
   @nrwl/web : 14.1.7
   @nrwl/workspace : 14.1.7
   typescript : 4.6.4
   rxjs : 6.6.7
   ---------------------------------------
   Community plugins:
         @nrwl/expo: 14.1.1

**Comment by mwarger** [comment]
I'll note that the intended command does indeed show up, so my current workaround is just copy-pasting the command and running it manually.

**Comment by mwarger** [comment]
> Interesting. At this point nx console isn't doing anything. It builds the command and sends it to vscode.
> 
> Have you tried killing vscode and starting again?

I have, multiple times.  I've tried reloading the window, re-installing the extension, restarting the app completely, and restarting my computer.

**Comment by Cammisuli** [comment]
Does it happen with the previous version of the extension?

**Comment by mwarger** [comment]
> Does it happen with the previous version of the extension?

I will give this a try.  Is there a way to downgrade within VSCode?

**Comment by Cammisuli** [comment]
Yea, you can. Find Nx Console in the extensions view, and click the gear icon. From there, select `Install another version...` and you can install previous versions there.

**Comment by mwarger** [comment]
I have tried the past 5 versions and none of them work.  I'm pretty confused about what's going on here...

**Comment by mwarger** [comment]
I think I just found the issue.  I can't believe I didn't think of this before, but I opened the VSCode developer tools and found a bunch of this in the console:

<img width="558" alt="image" src="https://user-images.githubusercontent.com/686823/174464836-de9d576a-49c2-4020-b0e9-62db5ece2cde.png">

Looks like some part of `fig`(https://fig.io/) was misconfigured.  It will try to automatically start itself when you open a new console, and I think this was choking out the commands that were being sent to the new terminal from NX.  I was able to click a `fig` option to re-init the install script and now I'm seeing the output from NX.  Thanks for your back and forth on this.
